plugin/__init__.py

```diff
@@ -918,6 +918,41 @@ def updateModelStyling(self, model):
         models.flush()
 
 
+    @util.api()
+    def findAndReplaceInModels(self, modelName, findText, replaceText, Front = True, Back = True, css = True):
+        if modelName == 'all_models':
+            ankiModel = self.collection().models.allNames()
+        else:
+            model = self.collection().models.byName(modelName)
+            if model is None:
+                raise Exception('model was not found: {}'.format(modelName))
+            ankiModel = [model]
+        updatedModels = 0    
+        for model in ankiModel:
+            model = self.collection().models.byName(modelName)
+            checkForText = False
+            if css and findText in model['css']:
+                checkForText = True
+                model['css'] = model['css'].replace(findText, replaceText) 
+            for tmpls in model.get('tmpls'):
+                for key, value in tmpls.items():
+                    if key in ('qfmt', 'afmt'):
+                        if Front and key == 'qfmt' and findText in value:
+                            checkForText = True
+                            tmpls[key] = value.replace(findText, replaceText)
+                        if Back and key == 'afmt' and findText in value:
+                            checkForText = True
+                            tmpls[key] = value.replace(findText, replaceText)                       
+            self.collection().models.save(model, True)
+            self.collection().models.flush()
+            if checkForText:
+                updatedModels += 1
+        if updatedModels == len(ankiModel):
+            return "Found and replaced in {0} models".format(updatedModels)
```

Comment:
Why return a formatted string? That can be annoying for caller that may want to use the value. Just return the count.